man/docker-build.1.md

```diff
@@ -229,18 +248,19 @@ as context. The Dockerfile at the root of the repository is used as
 Dockerfile. This only works if the GitHub repository is a dedicated
 repository.
 
-    docker build github.com/scollier/Fedora-Dockerfiles/tree/master/apache
+    docker build github.com/scollier/Fedora-Dockerfiles/tree/master/apache .
```

Comment:
The `.` here isn't needed, because the github-url takes it's place.

However, the example doesn't work (not your mistake), because a) that repo is moved, and b) docker expects the Dockerfile to be at the root of the repository. (here it's in a subdirectory `/apache/`)

I don't have an alternative repository at hand, but maybe you know one :smile: